node/node.go

```diff
@@ -268,6 +271,9 @@ func MakeFull(log logging.Logger, rootDir string, cfg config.Local, phonebookAdd
 		}
 	}
 
+	node.tracer = messagetracer.NewTracer().Init(cfg)
```

Comment:
it would be nice if you could pass the logger to the NewTracer so that we could stop using the logging.Base().